build.gradle

```diff
@@ -715,7 +715,21 @@ List<String> jacocoExclusions = [
         'org.opensearch.ad.AnomalyDetectorExtension',
         'org.opensearch.ad.transport.ADJobRunnerTransportAction*',
         'org.opensearch.ad.AnomalyDetectorProfileRunner',
-        'org.opensearch.ad.EntityProfileRunner'
+        'org.opensearch.ad.EntityProfileRunner',
+        'org.opensearch.ad.util.BulkUtil',
+        'org.opensearch.ad.common.exception.InternalFailure',
+        'org.opensearch.ad.cluster.HourlyCron',
+        'org.opensearch.ad.cluster.ADClusterEventListener',
+        'org.opensearch.ad.cluster.DailyCron',
+        'org.opensearch.ad.cluster.diskcleanup.IndexCleanup',
+        'org.opensearch.ad.cluster.diskcleanup.ModelCheckpointIndexRetention',
+        'org.opensearch.ad.transport.handler.AnomalyIndexHandler',
+        'org.opensearch.ad.transport.AnomalyResultTransportAction.RCFActionListener',
+        'org.opensearch.ad.transport.AnomalyResultRequest',
+        'org.opensearch.ad.transport.AnomalyResultAction',
+        'org.opensearch.ad.transport.CronNodeResponse',
+        'org.opensearch.ad.transport.CronResponse',
+        'org.opensearch.ad.transport.AnomalyResultResponse'
```

Comment:
Are all these exclusions part of this PR or just left over from a merge commit (i.e., fixed by rebase)?

Reply:
They are the part of this PR. Updating imports caused failure for few test cases, the functionality of which isn't implemented anyway.